var/spack/repos/builtin/packages/hicops/package.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hicops(Package):
+    """HiCOPS is a software framework for accelerating database peptide search
+    workflows on supercomputers. HiCOPS provided algorithm-independent
+    parallelizations and optimizations can be extended into new HPC database search
+    algorithms or scalably accelerate the existing ones.
+    """
+
+    homepage = "https://hicops.github.io/index"
+    git = "https://github.com/hicops/hicops.git"
+    maintainers = ['pcdslab', 'mhaseeb123', 'nessiecancode']
+
+    version('release', branch='release')
+    version('develop', branch='develop')
+
+    # Build Options
+    variant('USE_MPI', default='ON',
```

Comment:
Can you use lowercase for variants? That's what all the other packages do. In this specific case you need an `mpi` Boolean variant. Look for instance at the `mpi` variant in `hdf5`.

Reply:
done